docs/api/api/api.txt

```diff
@@ -1960,6 +1960,11 @@ POST /staging/<staging_workflow_project>/staging_projects/<staging_project>/copy
   Copy a staging project
 XmlResult: status_ok
 
+POST /staging/<staging_workflow_project>/staging_projects/merge
+  Merge one or multiple staging projects
```

Comment:
That's a rather strange wording. If I merge two staging projects, there is one afterwards. 'to merge == to combine or join together' (cambridge dictionary). But you don't merge staging projects here, you accept requests - and the staging projects number stays untouched - but they are empty afterwards.

synonyms for 'remove content' are:

1. clear
2. consume
3. deplete
4. discharge
5. drain
6. drink
7. dump
8. empty
9. escape
10. evacuate
11. exhaust
12. gut
13. leak
14. purge
15. run out
16. unload
17. vacate
18. decant
19. disgorge
20. ebb
21. eject
22. expel
23. leave
24. release
25. tap
26. unburden
27. void
28. flow out
29. make void
30. pour out
31. rush out
32. use up

Reply:
I like disgorge ;)

Makes nice sentenses 'You can please review X so I can disgorge the staging projects?' :)

Reply:
That was one of the open questions on my list. So far we called this feature Merge staging projects into Staging workflow main project' , but given that the overall_state required for this is called `acceptable` we might better call the action `accept` and the state `accepting`.